contextualize_eval.cpp

```diff
@@ -0,0 +1,88 @@
+#include "contextualize_eval.hpp"
+#include "ast.hpp"
+#include "eval.hpp"
+#include "backtrace.hpp"
+#include "to_string.hpp"
+#include "parser.hpp"
+
+namespace Sass {
+
+  Contextualize_Eval::Contextualize_Eval(Context& ctx, Eval* eval, Env* env, Backtrace* bt)
+  : Contextualize(ctx, env, bt), eval(eval)
+  { }
+
+  Contextualize_Eval::~Contextualize_Eval() { }
+
+  Selector* Contextualize_Eval::fallback_impl(AST_Node* n)
+  { return Contextualize::fallback_impl(n); }
```

Comment:
So i tried that and a lot more tests broke.  it's 2:48 am here  =) so i will need to sleep...but I welcome more feedback.  I'm looking at that one error statement that I removed.  The test was the closed issue 930 that was breaking when it is uncommented in.  That test, though, passes, when commented out (not exactly a litmus test but just saying).